<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/icon/creator.ico</normaloff>:/new/prefix1/icon/creator.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="newButton">
        <property name="toolTip">
         <string>New graph</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/new.ico</normaloff>:/new/prefix1/icon/new.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="openButton">
        <property name="toolTip">
         <string>Open a graph</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/open.ico</normaloff>:/new/prefix1/icon/open.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveButton">
        <property name="toolTip">
         <string>Save</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/save.ico</normaloff>:/new/prefix1/icon/save.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pointerButton">
        <property name="toolTip">
         <string>Pointer</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/pointer.ico</normaloff>:/new/prefix1/icon/pointer.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cycleButton">
        <property name="toolTip">
         <string>A Node</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/cycle.ico</normaloff>:/new/prefix1/icon/cycle.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="taskButton">
        <property name="toolTip">
         <string>Task</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/task.ico</normaloff>:/new/prefix1/icon/task.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="fictivetaskButton">
        <property name="toolTip">
         <string>Task Fictive</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/new/prefix1/icon/fictivetask.ico</normaloff>:/new/prefix1/icon/fictivetask.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="scaleSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Scale Slider</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>110</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointerButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pointerButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cycleButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cycleButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>taskButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>taskButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fictivetaskButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fictivetaskButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sceneScaleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openButtonClicked()</slot>
  <slot>newButtonClicked()</slot>
  <slot>saveButtonClicked()</slot>
  <slot>pointerButtonClicked()</slot>
  <slot>cycleButtonClicked()</slot>
  <slot>taskButtonClicked()</slot>
  <slot>fictivetaskButtonClicked()</slot>
  <slot>sceneScaleChanged(int)</slot>
 </slots>
</ui>
